src/pages/registration/resource_type_tab/sub_type_forms/research_data_types/RelatedResourceRow.tsx

```diff
@@ -1,53 +1,46 @@
 import RemoveCircleOutlineIcon from '@mui/icons-material/RemoveCircleOutline';
-import { Box, Button, Link, ListItem, Skeleton, Typography } from '@mui/material';
+import { Button, Link, ListItem, Skeleton, Typography } from '@mui/material';
 import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { Link as RouterLink } from 'react-router-dom';
 import { ConfirmDialog } from '../../../../../components/ConfirmDialog';
-import { Registration } from '../../../../../types/registration.types';
+import { RegistrationListItemContent } from '../../../../../components/RegistrationList';
+import { SearchListItem } from '../../../../../components/styled/Wrappers';
+import { RegistrationSearchItem } from '../../../../../types/registration.types';
 import { API_URL } from '../../../../../utils/constants';
 import { dataTestId } from '../../../../../utils/dataTestIds';
 import { useFetch } from '../../../../../utils/hooks/useFetch';
-import { getTitleString } from '../../../../../utils/registration-helpers';
-import { getRegistrationLandingPagePath } from '../../../../../utils/urlPaths';
 
-interface RelatedResourceRowRowProps {
+interface RelatedResourceRowProps {
   uri: string;
   removeRelatedResource: () => void;
 }
 
-export const RelatedResourceRow = ({ uri, removeRelatedResource }: RelatedResourceRowRowProps) => {
+export const RelatedResourceRow = ({ uri, removeRelatedResource }: RelatedResourceRowProps) => {
   const { t } = useTranslation();
   const isInternalRegistration = uri.includes(API_URL);
-  const [registration, isLoadingRegistration] = useFetch<Registration>({ url: isInternalRegistration ? uri : '' });
+  const [registration, isLoadingRegistration] = useFetch<RegistrationSearchItem>({
```

Comment:
blir ikke denne typen feil nå? Siden vi slår opp på `uri` skal vi få hele objektet, og ikke bare den ekspanderte versjonen fra søk
```suggestion
  const [registration, isLoadingRegistration] = useFetch<Registration>({
```